deploy/kubernetes/Makefile

```diff
@@ -16,6 +16,15 @@ KUSTOMIZE=elastic-agent-kustomize
 KUSTOMIZE_DEFAULT=elastic-agent-kustomize/default
 KUSTOMIZE_KSM_AUTOSHARDING=elastic-agent-kustomize/ksm-autosharding
 
+#PACKAGE Versions
+SYSTEM_VERSION=1.20.4
+K8S_VERSION=1.52.0
```

Comment:
Indeed not used. I just added them because I was thinking that we should update the standalone. For now I deleted them

But it is only the meta block here https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L35-L38, that actually is not used anywhere.